src/web/core/src/main/resources/GeoServerApplication_fr.properties

```diff
@@ -1051,3 +1050,21 @@ PublishedConfigurationPage.cqlUsesInvalidAttribute=L'expression CQL source de l'
 PublishedConfigurationPage.attributeInvalidConversion=Un probl�me a �t� d�tect� dans l'attribut {0}, impossible de convertir depuis le type d'origine {1} vers le type cible {2}
 PublishedConfigurationPage.attributeInvalidCQL=CQL invalude pour la source {0}. {1}
 PublishedConfigurationPage.multiAttributeSameName=Plusieurs d�finitions ont �t� trouv�es pour l'attribut de sortie {0}
+AbstractWMSStorePage.headerName=Nom de l'ent\u00EAte HTTP
```

Comment:
aside: These files can be setup to accept unicode if you change the file extension bit. Not sure if that works still with weblate?